ros_to_dhaiba:
 - from: /tf
   to:   tf

dhaiba_to_ros:
  - from: PARTICIPANT335589/myNote
    to:   /mynote
    type: geometry_msgs/Point
